source/crud/crud.rst

```diff
@@ -1,6 +1,14 @@
 .. role:: javascript(code)
   :language: javascript
 
+
+.. note::
+  This specification has been converted to Markdown and renamed to
+  `crud.md <crud.md>`_.  
+
+  Use the link above to access the latest version of the specification as the
+  current reStructuredText file will no longer be updated.
```

Comment:
Do you intend on leaving the old specs in place for outstanding PRs? We've already lost the RST for those that have been ported, so relying on the https://github.com/mongodb/specifications/releases/tag/v1 tag for git-blame purposes seem fine.

Reply:
Disregard. This was answered in https://github.com/mongodb/specifications/pull/1524#discussion_r1499386719.